### PR TITLE
integrate with mayhem

### PR DESCRIPTION
**Pull request by jbgreenh**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Target name (should be unique within the project)
+target: sndfilter
+
+# List of commands used to test the target
+cmds:
+
+    # Command used to start the target, "@@" is the input file
+    # (when "@@" is omitted Mayhem defaults to stdin inputs)
+    - cmd: /sndfilter @@ <reverb> <5> <longreverb1>
```
Comment (xansec) [inline Mayhemfile:12]: We do not currently support multiple commands in the Mayhemfile at the moment. Please either omit the extra commands, or split this Mayhemfile into multiple Mayhemfiles (see here: https://community.forallsecure.com/t/error-the-mayhem-for-code-scan-detected-the-mayhem-run-for-your-target-was-unsuccessful-and-multiple-docker-files-in-the-repository/350/5).

**Review by xansec — CHANGES_REQUESTED** [review]
We have changed support for corpora under `tests` to corpora under `testsuite`. Please change the name of this folder.

**Comment by jbgreenh** [comment]
updated

**Comment by xansec** [comment]
LGTM!

**Comment by xansec** [comment]
LGTM!

**Pull request merged**